src/views/TabView/TabView.js

```diff
@@ -22,12 +21,9 @@ class TabView extends React.PureComponent {
   };
 
   _renderScene = ({ route }) => {
-    const { screenProps } = this.props;
-    const childNavigation = this.props.childNavigationProps[route.key];
-    const TabComponent = this.props.router.getComponentForRouteName(
-      route.routeName
-    );
-
+    const { screenProps, descriptors } = this.props;
+    const descriptor = descriptors[route.key];
+    const TabComponent = descriptor.getComponent();
```

Comment:
the props passed in to `ResourceSavingSceneView` need to come from the navigationConfig config &mdash; `lazy`, `removeClippedSubviews`, `animationEnabled`, `swipeEnabled` etc aren't passed in directly as props to `TabView` anymore, they are provided via `props. navigationConfig`.

you can verify that this isn't working as expected by going to the stacks in tabs example on iOS and noticing that you can swipe back and forth between the tabs